engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/TypedRecordProcessor.java

```diff
@@ -23,32 +22,16 @@
   default void processRecord(final TypedRecord<T> record) {}
 
   /**
-   * @see #processRecord(TypedRecord, LegacyTypedResponseWriter, LegacyTypedStreamWriter, Consumer)
+   * @see #processRecord(TypedRecord, Consumer)
```

Comment:
as far as I can tell, this also closes #8002 So the TODO at the top can be removed

Reply:
Good point thanks 😊